docs/source/modules.rst

```diff
@@ -0,0 +1,7 @@
+src
```

Comment:
Please change this with something more meaningful, e.g. API docs (have a look at the Qibo docs).